inlong-tubemq/tubemq-client-twins/tubemq-client-go/client/heartbeat.go

```diff
@@ -99,36 +99,45 @@ func (h *heartbeatManager) consumerHB2Master() {
 	}
 
 	rsp, err := h.sendHeartbeatC2M(m)
-	if err != nil {
-		log.Errorf("consumer hb err %s", err.Error())
+	if err != nil || !rsp.GetSuccess() {
```

Comment:
When I use this version to verify, I found that the client did not respond to load balancing, and no longer registered Broker for consumption